indexer/ftypes_matcher.cpp

```diff
@@ -459,29 +419,51 @@ IsPisteChecker::IsPisteChecker() : BaseChecker(1 /* level */)
   m_types.push_back(classif().GetTypeByPath({"piste:type"}));
 }
 
-IsPoiChecker::IsPoiChecker() : BaseChecker(1 /* level */)
-{
-  string_view const poiTypes[] = {
-    "amenity",
-    "shop",
-    "tourism",
-    "leisure",
-    "sport",
-    "craft",
-    "man_made",
-    "emergency",
-    "office",
-    "historic",
-    "railway",
-    "highway",
-    "aeroway",
-    "healthcare",
+
+OneLevelPOIChecker::OneLevelPOIChecker() : ftypes::BaseChecker(1 /* level */)
+{
+  Classificator const & c = classif();
+
+  auto paths = {
+      "amenity",  "craft", "healthcare", "historic", "leisure", "office", "railway", "shop", "sport", "tourism",
+  };
+  for (auto const & path : paths)
```

Comment:
This reduces the number of asm instructions by two. Unfortunately, compilers are not smart :(
```suggestion
  for (auto const * path : paths)
```